usaspending_api/api_contracts/contracts/v2/agency/toptier_code/program_activity.md

```diff
@@ -0,0 +1,104 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# List Program Activity [/api/v2/agency/{toptier_code}/program_activity/]
+
+Returns a list of Program Activity in the Agency's appropriations for a single fiscal year
+
+## POST
+
++ Request (application/json)
+    + Schema
+
+            {
+                "$schema": "http://json-schema.org/draft-04/schema#",
+                "type": "number"
+            }
+    + Parameters
+        + `toptier_code`: 086 (required, number)
+            The toptier code of an agency (could be a CGAG or FREC) so only numeric character strings of length 3-4 are accepted.
+    + Attributes
+        + `fiscal_year` (optional, number)
+            The desired appropriations fiscal year. Defaults to the current FY.
+        + `filter` (optional, string)
```

Comment:
I wasn't helping with the `filter` path param on the filter tree endpoints, but would a `filter` body parameter  get confused with the `filters` param used in advanced search?

Reply:
`filter` is a common parameter or key for the app's API requests. There would be a benefit to reducing the usage of it when the term could be more specific for the request. However I don't believe it adds confusion.